metrics/sinks/stackdriver/stackdriver.go

```diff
@@ -42,85 +42,9 @@ const (
 	httpResponseCodeClientTimeout = -1
 )
 
-type StackdriverSink struct {
-	project               string
-	cluster               string
-	zone                  string
-	stackdriverClient     *sd_api.Service
-	minInterval           time.Duration
-	lastExportTime        time.Time
-	batchExportTimeoutSec int
-	initialDelaySec       int
-}
-
-type metricMetadata struct {
-	MetricKind string
-	ValueType  string
-	Name       string
-}
-
 var (
-	// Known metrics metadata
-
-	cpuReservedCoresMD = &metricMetadata{
-		MetricKind: "GAUGE",
-		ValueType:  "DOUBLE",
-		Name:       "container.googleapis.com/container/cpu/reserved_cores",
-	}
-
-	cpuUsageTimeMD = &metricMetadata{
-		MetricKind: "CUMULATIVE",
-		ValueType:  "DOUBLE",
-		Name:       "container.googleapis.com/container/cpu/usage_time",
-	}
-
-	uptimeMD = &metricMetadata{
-		MetricKind: "CUMULATIVE",
-		ValueType:  "DOUBLE",
-		Name:       "container.googleapis.com/container/uptime",
-	}
-
-	networkRxMD = &metricMetadata{
-		MetricKind: "CUMULATIVE",
-		ValueType:  "INT64",
-		Name:       "container.googleapis.com/container/network/received_bytes_count",
-	}
-
-	networkTxMD = &metricMetadata{
-		MetricKind: "CUMULATIVE",
-		ValueType:  "INT64",
-		Name:       "container.googleapis.com/container/network/sent_bytes_count",
-	}
-
-	memoryLimitMD = &metricMetadata{
-		MetricKind: "GAUGE",
-		ValueType:  "INT64",
-		Name:       "container.googleapis.com/container/memory/bytes_total",
-	}
-
-	memoryBytesUsedMD = &metricMetadata{
-		MetricKind: "GAUGE",
-		ValueType:  "INT64",
-		Name:       "container.googleapis.com/container/memory/bytes_used",
-	}
-
-	memoryPageFaultsMD = &metricMetadata{
-		MetricKind: "CUMULATIVE",
-		ValueType:  "INT64",
-		Name:       "container.googleapis.com/container/memory/page_fault_count",
-	}
-
-	diskBytesUsedMD = &metricMetadata{
-		MetricKind: "GAUGE",
-		ValueType:  "INT64",
-		Name:       "container.googleapis.com/container/disk/bytes_used",
-	}
-
-	diskBytesTotalMD = &metricMetadata{
-		MetricKind: "GAUGE",
-		ValueType:  "INT64",
-		Name:       "container.googleapis.com/container/disk/bytes_total",
-	}
+	useOldResourceModel = true
+	useNewResourceModel = false
```

Comment:
Why isn't it a part of the `StackdriverSink` struct?

Reply:
Moved there